dynawo/3rdParty/nicslu/nicslu-chain.sh

```diff
@@ -68,16 +67,16 @@ install_nicslu() {
     cd $BUILD_DIR/$NICSLU_DIR
     if [ "$DYNAWO_LIBRARY_TYPE" = "SHARED" ]; then
       if [ "${BUILD_TYPE}" = "Debug" ]; then
-        make -j $NB_PROCESSORS_USED shared DEBUGFLAG="-g" OPTIMIZATION="-O0" || error_exit "Error while building Nicslu"
+        make shared DEBUGFLAG="-g" OPTIMIZATION="-O0" CC=$DYNAWO_C_COMPILER || error_exit "Error while building Nicslu"
```

Comment:
Why aren't we using the multi threading anymore?

Reply:
Nicslu makefiles are not done well to support make -j